integreat_cms/cms/views/external_calendars/external_calendar_form_view.py

```diff
@@ -95,6 +95,9 @@ def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
                 )
         else:
             external_calendar_form.instance.region = self.request.region
+            if not external_calendar_form.instance.created_by:
+                external_calendar_form.instance.created_by = request.user
+            external_calendar_form.instance.last_changed_by = request.user
```

Comment:
Nit: I wonder if it would be better to make use of `additional_instance_attributes` here, and only pass the current user. To me it seems like it should be the task of the form (and not the view) to correctly set the `created_by` and `last_changed_by` fields

Reply:
Done :white_check_mark: